src/mapeo-manager.js

```diff
@@ -533,6 +651,15 @@ export class MapeoManager extends TypedEmitter {
     await this.#mediaServer.stop()
   }
 
+  async startLocalPeerDiscovery() {
+    return this.#localDiscovery.start()
+  }
+
+  /** @type {LocalDiscovery['stop']} */
+  async stopLocalPeerDiscovery(opts) {
+    return this.#localDiscovery.stop(opts)
+  }
```

Comment:
your reasoning for new methods makes sense, but should these still be called in `manager.start()` and `manager.stop()`?

Reply:
alternatively, wondering if renaming start and stop to be more specific would make sense?

Reply:
Yes I'd been meaning to open a PR to call the existing methods "startMediaServer" and "stopMediaServer" or something like that

Reply:
created an issue for it: https://github.com/digidem/mapeo-core-next/issues/391